src/api/python/_api.py

```
@@ -702,74 +708,75 @@ def query_xfer_backend(self, handle: nixl_xfer_handle) -> str:
             if backendH == b_handle
         )
 
-    """
-    @brief  Releases a transfer handle, which internally frees the memory used for the handle.
-            If the transfer is active, NIXL will attempt to cancel it.
-            If it cannot be canceled, an error will be returned and the handle will not be freed.
-
-    @param handle Handle to the transfer operation from initialize_xfer or make_xfer.
-    """
-
     def release_xfer_handle(self, handle: nixl_xfer_handle):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `release_xfer_handle`

Add return type annotation: `None`

(ANN202)